lib/discourse_reactions/migration_report.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module DiscourseReactions
+  # TODO (martin) Remove this once we have rolled out reactions like sync more widely.
+  class MigrationReport
+    def self.run(previous_report_data: {}, print_report: true)
+      report_data = {
+        topic_user_liked: TopicUser.where(liked: true).count,
+        user_actions_liked: UserAction.where(action_type: UserAction::LIKE).count,
+        user_actions_was_liked: UserAction.where(action_type: UserAction::WAS_LIKED).count,
+        post_action_likes: PostAction.where(post_action_type_id: PostActionType.types[:like]).count,
+        post_like_count_total: Post.sum(:like_count),
+        topic_like_count_total: Topic.sum(:like_count),
+        user_stat_likes_given_total: UserStat.sum(:likes_given),
+        user_stat_likes_received_total: UserStat.sum(:likes_received),
+        given_daily_like_total: GivenDailyLike.sum(:likes_given),
+        badge_count: UserBadge.count,
+        reactions_total: DiscourseReactions::ReactionUser.count,
+      }
```

Comment:
Similar to previous comment, report data can be `class_attribute`

Reply:
I would prefer to leave this one as is -- it will be much more helpful for debugging this way